src/bootctl/bootctl.c

```diff
@@ -236,6 +236,7 @@ static int help(int argc, char *argv[], void *userdata) {
                "\n%3$sBoot Loader Interface Commands:%4$s\n"
                "  set-default ID       Set default boot loader entry\n"
                "  set-oneshot ID       Set default boot loader entry, for next boot only\n"
+               "  set-sysfail ID       Set boot loader entry to be used for a system failure\n"
```

Comment:
"for" → "in case of"

Reply:
Rephrased it,  changed "to be used"-> "used"